pkg/network/ebpf/c/prebuilt/http.c

```diff
@@ -204,7 +205,7 @@ int uretprobe__SSL_read(struct pt_regs* ctx) {
 
     skb_info_t skb_info = {0};
     __builtin_memcpy(&skb_info.tup, t, sizeof(conn_tuple_t));
-    http_process(buffer, &skb_info, skb_info.tup.sport);
+    http_process(buffer, &skb_info, skb_info.tup.sport, LIBSSL);
```

Comment:
I think we probably want `HTTP|LIBSSL`?
Also, do you think it would make sense to distinguish between OpenSSL, GnuTLS and etc?

Reply:
HTTP is added based on socket_filter  (need http_monitor = true)
LIBSSL based on libssl (need https_monitor = true)
if you
 * wget http://www you got already both tags   **=>** we have probably an issue here as we report 2 times the requests/responses int the HTTPStats; not sure (may ssl passthrough) why SSL_read/write are called
 * wget https://www LIBSSL only